daemon/logger/jsonfilelog/jsonfilelog.go

```diff
@@ -38,36 +42,123 @@ func New(ctx logger.Context) (logger.Logger, error) {
 	if err != nil {
 		return nil, err
 	}
+	var capval int64
+	capacity, ok := ctx.Config["max-size"]
+	if !ok {
```

Comment:
Instead of doing the `if; else` can we just set `capval = -`; then do `if !ok { ..}`?